src/components/AskQuestion/AskQuestion.vue

```diff
@@ -194,9 +195,10 @@ export default {
         });
         this.currentQuestion.questionTitle = "";
         this.currentQuestion.questionDescription = "";
-      }
+      }else{
+        window.alert("Bitte logge dich ein!");
```

Comment:
It looks like this message also appears when the validation fails.
Please Split the validation check from the login check. Separation of concerns.

Reply:
fixed

Reply:
done